### DISCUSSION TITLE
remix升级，似乎和之前有差距了，有人实践过吗？

### DISCUSSION THREAD

**Discussion by guohao655** [AmazingAng/WTF-Solidity#494] [Q&A] [answered]
<img width="647" alt="image" src="https://github.com/AmazingAng/WTF-Solidity/assets/54994470/13896ec6-c599-4bf7-9522-d7b0e83d898f">
<img width="285" alt="image" src="https://github.com/AmazingAng/WTF-Solidity/assets/54994470/db81a7f5-19b8-4daf-b49b-27c5ce151be3">

就是现在不会调用fallback函数，而是直接报出没有方法

**Comment by AmazingAng** [comment, marked as answer]
根据报错来看，你的合约中没定义fallback函数，可能你没复制完整合约。
完整合约: https://github.com/AmazingAng/WTF-Solidity/blob/main/22_Call/Call.sol
